package.json

```diff
@@ -64,7 +64,8 @@
       }
     },
     "hooks": {
-      "init": "./src/hooks/init/twilio-api"
+      "init": "./src/hooks/init/twilio-api",
+      "post-api-plugin": "./src/hooks/post-api-plugin/plugin-verification"
```

Comment:
So, this is a custom hook. Here are the ones that oclif provides: https://oclif.io/docs/hooks#lifecycle-events

`prerun` is too late since the command has already been found. If we wanted to remove the plugin from the command tree, it would be too late.

Since we have to use a custom hook, it has to be initiated somewhere in our code, and it has to happen before the command begins execution. And the hook has to fire *after* our API plugin has been fully loaded (hook processor are executed async so if we add another `init` hook processor there's no guarantee it will be run after `twilio-api` completes).

So I decided to make it very explicit that this hook fires *after* the API plugin is _installed_.

Another option is to add a new `post-init` hook to the oclif source, but not sure if they'd be supportive of this or how long it would take.

Reply:
I think the adding a `post-init` hook to the oclif source would be a nice to have, but not needed in the near term. 

What are your thoughts on the naming convention of `post-api-plugin` for the custom hook?

Reply:
I chose it to make it explicit: _this hook will fire after the API plugin is run_. It also passes the API plugin instance through the hook.